docs/content/preview/yugabyte-platform/anywhere-automation/anywhere-cli.md

```diff
@@ -0,0 +1,273 @@
+---
+title: yba CLI
+headerTitle: YugabyteDB Anywhere CLI
+linkTitle: yba CLI
+description: Use YugabyteDB Anywhere CLI to create and manage universes.
+headcontent: Install yba, configure default settings, and set up autocompletion
+earlyAccess: /preview/releases/versioning/#feature-availability
+menu:
+  preview_yugabyte-platform:
+    parent: anywhere-automation
+    identifier: anywhere-cli
+    weight: 50
+type: docs
+rightNav:
+  hideH4: true
+---
+
+The [YugabyteDB Anywhere (YBA) Command Line Interface](https://github.com/yugabyte/yugabyte-db/tree/master/managed/yba-cli) (yba) is an open source tool that enables you to interact with YBA accounts using commands in your command-line shell. With minimal configuration, you can start running commands from the command line that implement functionality equivalent to that provided by the browser-based YBA interface.
+
+## Install yba
```

Comment:
Install CLI.

Throughout this page, I would recommend calling the yba CLI and "the CLI". The term "yba" is overloaded, as it also refers to the YBA software itself.